src/init.sh

```diff
@@ -18,12 +18,21 @@ function init_kw()
     exit 0
   fi
 
+  if [[ -f "$PWD/$name" ]]; then
+    if [[ "$*" =~ --?f(orce)? || $(ask_yN "$name already exists, do you wish to overwrite it?") =~ '1' ]]; then
+      mv "$PWD/$name" "$PWD/$name.old"
+    else
+      say 'Initialization aborted!'
+      exit 0
```

Comment:
Add a test that validates this condition.

Reply:
Done!